build.gradle

```diff
@@ -52,8 +52,8 @@ dependencies {
     implementation 'com.azure:azure-identity:1.16.0'
 
     implementation 'com.azure.spring:spring-cloud-azure-starter-active-directory:5.22.0'
-    implementation 'com.h2database:h2'
-//    implementation 'org.postgresql:postgresql'
+    //implementation 'com.h2database:h2'
+    implementation 'org.postgresql:postgresql'
```

Comment:
please remove following at line 70
testImplementation 'org.postgresql:postgresql'
I added that because now we are using h2 but once switched back then no need of this testImplementation